### PR TITLE
Add mvn and npm support

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
This PR adds support next package managers:
- mvn (Apache Maven)
- npm (Default node package manager)
- yarn (Yarn)

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/20080

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Container wrapper for Eclipse Dash License Tool
 
 This is a container wrapper for [The Eclipse Dash License Tool](https://github.com/eclipse/dash-licenses) that allows easily to generate dependencies files with container image without the need to compile `dash-licenses` jar.
+It supports next package-manager:
```
Comment (sleshchenko) [inline README.md:4]:
```suggestion
It supports the following package managers:
```

File: README.md [hunk 2]
```diff
@@ -34,3 +38,10 @@ docker run --rm -t \
        -v ${PWD}/:/workspace/project  \
        quay.io/che-incubator/dash-licenses:next --check
 ```
+This command copies all files from the temporary directory. It returns a non-zero exit code if any of the dependencies are restricted to use.
```
Comment (sleshchenko) [inline README.md:41]: I would refactor the following section in the following way:
<pre>
## Usage

### Update dependency info

The following command generates dependencies info of a project and then checks all found dependencies. It returns a non-zero exit code if any of them are restricted to use.
```sh
docker run --rm -t \
       -v ${PWD}/:/workspace/project  \
       quay.io/che-incubator/dash-licenses:next
```
As the result this command creates next files: 
- `prod.md` with the list of production dependencies;
- `dev.md` which contains only build and test dependencies;
- `problems.md` will be created if some dependencies are not covered with CQ, unnecessary excludes present, etc. 

### Check dependencies

If you just need to verify that all dependencies satisfy IP requirements, use the `--check` flag, like the following
```sh
docker run --rm -t \
       -v ${PWD}/:/workspace/project  \
       quay.io/che-incubator/dash-licenses:next --check
```

So, this command doesn't create any new files in the project directory (except a temporary one) but checks if the dependencies info is up-to-date and then validates all found dependencies. It returns a non-zero exit code if any of the dependencies are restricted to use. 

### Debug

If you need all the generated files including logs and intermediate, use `--debug` flag:

```sh
docker run --rm -t \
       -v ${PWD}/:/workspace/project  \
       quay.io/che-incubator/dash-licenses:next --debug
```

This command copies all files from the temporary directory. It returns a non-zero exit code if any of the dependencies are restricted to use.
```
</pre>

**Review by sleshchenko — APPROVED** [review]
I haven't tested well, but overall LGTM

File: README.md (hunk 2 above)
Comment (olexii4) [reply README.md]: Thanks. I have fixed it.

File: README.md (hunk 1 above)
Comment (olexii4) [reply README.md]: Thanks. I have fixed it.

**Review by akurinnoy — APPROVED** [review]

**Pull request merged**